pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -639,7 +639,7 @@ public Optional<ScanError> updateKitLabel(KitRequestShipping kitRequestShipping)
                     dbVals.resultValue = new ScanError(kitRequestShipping.getDdpLabel(), "dsm_kit_id "
                             + kitRequestShipping.getDsmKitId() + " does not exist or already has a Kit Label");
                 } else {
-                    logger.info("Updated kitRequests for pt w/ shortId " + kitRequestShipping.getHruid());
+                    logger.info(String.format("Updated kitRequests for pt w/ shortId %s", kitRequestShipping.getHruid()));
```

Comment:
sonar fixes

Reply:
A better fix is to use `{}` provided by the logger class as direct string replacement.